setup.py

```diff
@@ -83,6 +83,7 @@ def check_setuptools_features():
     'aiohttp~=2.0',
     'python-rapidjson-schema==0.1.1',
     'statsd==3.2.1',
+    'yarl>=0.11,<1.0',
```

Comment:
just to share a different approach we could use:the `~=` operator:

```python
'yarl~=0.11',
```

see https://packaging.python.org/tutorials/distributing-packages/#semantic-versioning-preferred:

> Adopting this approach as a project author allows users to make use of “compatible release” specifiers, where `name ~= X.Y` requires at least release `X.Y`, but also allows any later release with a matching MAJOR version.